src/QMCWaveFunctions/BsplineFactory/HybridRepCenterOrbitals.h

```diff
@@ -708,7 +708,7 @@ class HybridRepCenterOrbitals
   }
 
   template<class BSPLINESPO>
-  friend class HybridRepSetReader;
+  friend struct HybridRepSetReader;
```

Comment:
We really prefer class. So even just fix as `class{ public:` is better than `struct`

Reply:
updated

Reply:
Thanks Steve. There are plenty of structs that should be classes.